x-pack/plugin/esql/qa/testFixtures/src/main/resources/voyager.csv-spec

```diff
@@ -0,0 +1,101 @@
+# ----------------------------------------------------------------------------------------------------------------------
+t1
+# ----------------------------------------------------------------------------------------------------------------------
```

Comment:
The big separator comments are likely leftovers. One is probably fine, but two feels like a bit much.

Reply:
It's probably worth sticking this into `stats_all_first_all_last.csv-spec` rather than `voyager.csv-spec`. Sticking them all into the same file makes it easier to track them. We don't have rule about how to organize these, but it's probably better to put all of the tests for these functions into the same file.